### PR TITLE
polish conv2d api 2.0

### PR DESCRIPTION
**Pull request by GaoWei8**
更新conv2d、conv2d_transpose、depthwise_conv2d的api调用脚本和json配置

### CODE REVIEW

**Review by Xreki — DISMISSED** [review]
conv2d没有问题了，但是conv2d_transpose和depthwise_conv2d.py中都是继承了Conv2dConfig，可能会挂，需要修复下。

File: api/tests_v2/configs/conv2d_transpose.json [hunk 1]
```diff
@@ -106,6 +100,11 @@
             "shape": "[-1L, 512L, 1L, 1L]",
             "type": "Variable"
         },
+        "weight": {
```
Comment (Xreki) [inline api/tests_v2/configs/conv2d_transpose.json:103]: filter_size和num_filters没删

File: api/tests_v2/configs/conv2d_transpose.json [hunk 2]
```diff
@@ -189,6 +185,11 @@
             "shape": "[1L, 1L, 80L, 63L]",
             "type": "Variable"
         },
+        "weight": {
+            "dtype": "float32",
+            "shape": "[1L, 1L, 3L, 32L]",
+            "type": "Variable"
+        },
```
Comment (Xreki) [inline api/tests_v2/configs/conv2d_transpose.json:192]: filter_size和num_filters没删

File: api/tests_v2/configs/depthwise_conv2d.json [hunk 3]
```diff
@@ -96,6 +90,11 @@
             "shape": "[-1L, 128L, 257L, 257L]",
             "type": "Variable"
         },
+        "weight": {
+            "dtype": "float32",
+            "shape": "[128L, 1L, 3L, 3L]",
+            "type": "Variable"
+        },
```
Comment (Xreki) [inline api/tests_v2/configs/depthwise_conv2d.json:97]: filter_size和num_filters没删

File: api/tests_v2/configs/conv2d_transpose.json (hunk 1 above)
Comment (GaoWei8) [reply api/tests_v2/configs/conv2d_transpose.json:103]: Done.

File: api/tests_v2/configs/conv2d_transpose.json (hunk 2 above)
Comment (GaoWei8) [reply api/tests_v2/configs/conv2d_transpose.json:192]: Done.

File: api/tests_v2/configs/depthwise_conv2d.json (hunk 3 above)
Comment (GaoWei8) [reply api/tests_v2/configs/depthwise_conv2d.json:97]: Done.

**Review by Xreki — APPROVED** [review]

**Pull request merged**